content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,157 @@
+Title: Where to incorporate if you are a EU citizen
+Date: 2016-07-10
+Short_summary: We have lots of options as to where incorporate as EU citizens, let's look at them.
```

Comment:
this bit i'm not even sure where it appears to be honest, maybe the small twitter summary etc?